analyzers/tests/SonarAnalyzer.UnitTest/Rules/UnusedPrivateMemberTest.Fields.cs

```diff
@@ -128,5 +128,34 @@ public object Method2()
     }
 }
 ").Verify();
+
+        [TestMethod]
+        public void UnusedPrivateMember_Fields_StructLayout() =>
+            builder.AddSnippet("""
```

Comment:
I prefer this alignment as well, the problem is that all the other snippets in these UTs are aligned from the very left (no tabs). Either change everything else or just comply with what is already there IMO